README.md

```diff
@@ -22,11 +22,54 @@ The goal of this package is to offer almost all the features available on the we
 
 3. Login with your credentials and move to the profile section.
 
-4. Click on `Get your Auth Token` to get the authentication token & download it as a JSON file.
+4. To setup the auth token, run the following command:
```

Comment:
This command will work only once `evalai` package is setup. Please move it after the step 4.